contracts/ethereum/script/Deploy.s.sol

```diff
@@ -10,15 +10,15 @@ contract Deploy is Script {
         uint256 deployerPrivateKey = vm.envUint("ETHEREUM_PRIVATE_KEY");
         vm.startBroadcast(deployerPrivateKey);
 
-        address snMessagingAddress = vm.envAddress("STARKNET_MESSAGING_ADDRESS");
-        uint256 snEscrowAddress = 0x0; // this value is set in a call to the smart contract, once deployed
-        uint256 snClaimPaymentSelector = 0x0; // this value is set in a call to the smart contract, once deployed
+        address STARKNET_MESSAGING_ADDRESS = vm.envAddress("STARKNET_MESSAGING_ADDRESS");
+        uint256 STARKNET_CLAIM_PAYMENT_SELECTOR = vm.envUint("STARKNET_CLAIM_PAYMENT_SELECTOR");
         address marketMaker = vm.envAddress("MM_ETHEREUM_WALLET_ADDRESS");
```

Comment:
I would suggest renaming variable to MARKET_MAKER or MM_ETHEREUM_WALLET_ADDRESS for consistency with new naming